### PR TITLE
fix: db connector with schema

### PR DESCRIPTION
**Pull request by saitofun**

### CODE REVIEW

File: pkg/depends/conf/postgres/postgres.go [hunk 1]
```diff
@@ -76,16 +76,14 @@ func (e Endpoint) UseSlave() sqlx.DBExecutor {
...
-	return e.conn(e.masterURL(), false)
-}
-
 func (e *Endpoint) conn(url string, readonly bool) (*sqlx.DB, error) {
 	connector := &postgres.Connector{
 		Host:  url,
 		Extra: e.Master.Param.Encode(),
 	}
+	if e.Database != nil {
+		connector.MustSchema = e.Database.Schema
```
Comment (Liuhaai) [inline pkg/depends/conf/postgres/postgres.go:85]: where is the source of `e.Database.Schema` from?

File: cmd/srv-applet-mgr/config/config.yml.template [hunk 2]
```diff
@@ -23,7 +23,7 @@ SRV_APPLET_MGR__MqttBroker_RetainPublish: "false"
 SRV_APPLET_MGR__MqttBroker_Retry_Interval: 3s
 SRV_APPLET_MGR__MqttBroker_Retry_Repeats: "3"
 SRV_APPLET_MGR__MqttBroker_Server: mqtt://127.0.0.1:1883
-SRV_APPLET_MGR__MqttBroker_Timeout: 0s
+SRV_APPLET_MGR__MqttBroker_Timeout: 10s
```
Comment (Liuhaai) [inline cmd/srv-applet-mgr/config/config.yml.template:26]: revert in this pr

**Comment by Liuhaai** [comment]
what issue does this pr fix

File: pkg/types/wasm/wasm_config_database.go [hunk 3]
```diff
@@ -23,11 +23,7 @@ func (s *Schema) ConfigType() enums.ConfigType {
 }
 
 func (s *Schema) WithContext(ctx context.Context) context.Context {
-	db, err := types.MustWasmDBExecutorFromContext(ctx).NewConnection()
-	if err != nil {
-		panic(err)
-	}
-
+	db := types.MustWasmDBExecutorFromContext(ctx)
```
Comment (Liuhaai) [inline pkg/types/wasm/wasm_config_database.go:26]: https://github.com/machinefi/w3bstream/pull/400/files#diff-555c21773fba96fe0e5b1597f376ba379c070864b7ca39084d45a0b15957db52R33-R35
is useless in the pr

File: pkg/depends/conf/postgres/postgres.go (hunk 1 above)
Comment (saitofun) [reply pkg/depends/conf/postgres/postgres.go:85]: in models.database.go where they are defined

File: cmd/srv-applet-mgr/config/config.yml.template (hunk 2 above)
Comment (saitofun) [reply cmd/srv-applet-mgr/config/config.yml.template:26]: The default value is assigned in `DefaultSetter` interface.
conf/mqtt.go:27

File: pkg/types/wasm/wasm_config_database.go (hunk 3 above)
Comment (saitofun) [reply pkg/types/wasm/wasm_config_database.go:26]: yes, it will be removed in pr #403

**Pull request merged**